DependencyInjection/Configuration.php

```diff
@@ -35,6 +35,7 @@ public function getConfigTreeBuilder()
                         ->scalarNode('recipient')->isRequired()->cannotBeEmpty()->end()
                         ->scalarNode('sender')->isRequired()->cannotBeEmpty()->end()
                         ->scalarNode('subject')->isRequired()->cannotBeEmpty()->end()
+                        ->booleanNode('send_on_warning')->defaultTrue()->end()
```

Comment:
I also agree that its most likely that it will boil down to just skipping on warnings. Just seemed weird to have such a specific setting. That being said, if we ever do change this for some unforeseen reason, we can do some BC logic in the Configuration class.